### ISSUE TITLE
[Android,iOS] Notification hide Seek buttons

### ISSUE THREAD

**Issue by valentin-debris** [martijn00/XamarinMediaManager#514]
## 🚀 Feature Requests

Hi @martijn00, I want to remove the seek buttons for the notification. The streams I use for the mediaplayer are live stream, so they should not be allowed to use the seek button.

### Contextualize the feature
I know you added useNavigationActions and based on this link (https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/ui/PlayerNotificationManager.html), this can be achieve by adding rewindIncrementMs and fastForwardIncrementMs

### Platforms affected (mark all that apply)
- [x] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by martijn00** [comment]
I'm not sure what you what you mean. This is already available? https://github.com/martijn00/XamarinMediaManager/blob/develop/MediaManager/Notifications/INotificationManager.cs#L7

**Comment by valentin-debris** [comment]
@martijn00 
The navigation controls are to move between differents url. The seek buttons are to move inside one url (add/remove 10 sec for example). 
In fact, I only want the Pause(/Play) & Stop button inside the notification..

**Comment by martijn00** [comment]
Then you can set the StepSize: https://github.com/martijn00/XamarinMediaManager/blob/develop/MediaManager/Playback/IPlaybackManager.cs#L22

**Comment by valentin-debris** [comment]
@martijn00, my bad, I didn't find it !
But I tried and it's not working, I still have the arrow with the following code :

```
CrossMediaManager.Current.StepSize = TimeSpan.Zero;
await CrossMediaManager.Current.Play("https://storage.googleapis.com/exoplayer-test-media-0/play.mp3");
```

**Comment by martijn00** [comment]
I think there is a slight problem that this property only gets picked up once, so you need to set it before calling anything else. Probably even before calling Init().

I am looking into improving that in a later release.

**Comment by valentin-debris** [comment]
@martijn00 ,

Perfect, placing it before Init() fixe the issue for Android. I will check on iOS later.

**Comment by valentin-debris** [comment]
@martijn00 , this is not working for iOS, I still have the arrows

**Comment by martijn00** [comment]
Hmm, i think that this is not the full solution then: https://github.com/martijn00/XamarinMediaManager/blob/develop/MediaManager/Platforms/Apple/Notifications/NotificationManager.cs#L91

Maybe i need to remove the targets as well. You could try and add that to your own code for now.

**Comment by valentin-debris** [comment]
@martijn00 , Ok. 
But what do you means by "add on your own code" ? What and where ? 
I added the block in the else in AppDelegate but not working, either setting NotificationManager.ShowNavigationControls to false.

Thanks

**Comment by valentin-debris** [comment]
@martijn00 , in fact, the solution for #515 solves this problem (but I guess it's still a bug !).

Solution to hide seek buttons :
```
CrossMediaManager.Current.StepSize = TimeSpan.Zero;
CrossMediaManager.Current.Init();
CrossMediaManager.Current.NotificationManager.ShowPlayPauseControls = true;
CrossMediaManager.Current.NotificationManager.ShowNavigationControls = false; //In case you want to remove the next/prev song too
```

**Comment by martijn00** [comment]
I've released a fix in 0.6.9 to make sure StepSize is updated on Android. Can you test?

This means you should now call it after Init, since Init should always be the first thing.
